pkg/config/cloud_lb_config_test.go

```diff
@@ -0,0 +1,55 @@
+package config
```

Comment:
NIT: all this content rather goes to `node_test.go` file as it's only 2 test cases, so no need to have a dedicated file for them